paseos/visualization_example/example_interactive_plot.py

```diff
@@ -0,0 +1,42 @@
+import sys
+
+sys.path.append("../..")
+
+from paseos import ActorBuilder, SpacecraftActor
+from paseos.visualization.space_animation import SpaceAnimation
+
+
+import pykep as pk
+import matplotlib.pyplot as plt
+from utils import test_setup
+
+
+def interactive_plot():
```

Comment:
same as other example, lacks any explanation

Reply:
I removed the util and added description.